CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing
+
+Contributing to this repository, we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within the project.
+
+## Issue tracker
+
+Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the [issue tracker](https://github.com/ansible-collections/community.mysql/issues).
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction initially and potentially save a lot of time and effort.
+Also somebody can already start discussing or working on implementing the same or similar idea,
+so you can cooperate to create a better solution together.
+
+## Open pull requests
+
+Look through currently [open pull requests](https://github.com/ansible-collections/community.mysql/pulls).
+You can help by reviewing them. There can be good pull requests which are not merged only because there are a lack of reviews. Also there can be worth, reviewed, but abandoned ones, which you could politely ask the original authors to complete yourself.
+And it is always worth saying that good reviews are often more valuable than pull requests themselves.
+
+## Looking for an idea to implement
+
+First, see the paragraphs above.
+
+If you came up with a new feature, it is always worth creating an issue
+before starting to write code to discuss the idea with the community first.
+
+## Step-by-step guide how to get into development quickly.
+
+We assume you use Linux as a work environment (you can use a virtual machine as well).
+
+1. Install ``docker``, launch it.
+
+2. Clone the [ansible-core](https://github.com/ansible/ansible) repository:
+```bash
+git clone https://github.com/ansible/ansible.git
+```
```

Comment:
I would skip these steps. Developing collections can also be done with a regular Ansible 2.9, 2.10 or 3 install.

Reply:
I sometimes notice that when i forget to run `source hacking/env-setup`, tests against collections don't work

Reply:
But in case you installed ansible regularly (as most people do), that isn't needed.

Reply:
it's not needed but wouldn't it be better to use a version from devel? I don't insist, just interesting to know

Reply:
My main argument against it would be that it makes it harder to set everything up :) If you use the regular version of Ansible(-base/-core) you have installed anyway, most things will just work the same way.

Reply:
Ok:) I wouldn't like to break the fragile equilibrium reached:) So, would be perfect if you remove it as suggested and make the document consistent in other parts (because I've never used the default installation). Do you have a bit time for that?